### PR TITLE
Feat/checkout

### PR DESCRIPTION
**Pull request by arwassa**
- Updated shopping cart styling
 - Made it possible to place orders

### CODE REVIEW

File: frontend/src/app/app-routing.module.ts [hunk 1]
```diff
@@ -10,6 +10,7 @@ import { DealsComponent } from "./deals/deals.component";
 import { ProductContainerComponent } from "./product-container/product-container.component";
 import { ShoppingCartComponent } from "./shopping-cart/shopping-cart.component";
 import { ProductDetailsComponent } from "./product-details/product-details.component";
+import { CheckoutComponent } from './checkout/checkout.component';
```
Comment (DeimosA) [inline frontend/src/app/app-routing.module.ts]: What does the checkout component do?

File: frontend/src/app/shopping-cart-list/shopping-cart-list.component.html [hunk 2]
```diff
@@ -1,48 +1,51 @@
...
+        <div>
+          <a [routerLink]="['/product', item._prodId]" class="black-text">
+            {{product(item).name}}
+          </a>
         </div>
-      </a>
-    </li>
+      </td>
+      <td class="cart-td-center" *ngIf="productLoaded(item)">
+        <button (click)="updateQty($event, item, 1)" class="btn-flat waves-effect waves-light cart-btn-small"><i class="material-icons cart-icon-btn">add</i></button>
```
Comment (DeimosA) [inline frontend/src/app/shopping-cart-list/shopping-cart-list.component.html]: Increment on right side

File: frontend/src/app/shopping-cart-list/shopping-cart-list.component.html [hunk 3]
```diff
@@ -1,48 +1,51 @@
...
+        <span>{{item.quantity}}</span>
+        <button (click)="updateQty($event, item, -1)" class="btn-flat waves-effect waves-light cart-btn-small"><i class="material-icons cart-icon-btn">remove</i></button>
+      </td>
+      <td class="cart-td-right" *ngIf="productLoaded(item)">{{item.quantity * product(item).price.toFixed(2)}}kr</td>
```
Comment (DeimosA) [inline frontend/src/app/shopping-cart-list/shopping-cart-list.component.html]: total calculated price must be toFixed

File: frontend/src/app/shopping-cart-list/shopping-cart-list.component.ts [hunk 4]
```diff
@@ -56,4 +77,22 @@ export class ShoppingCartListComponent implements OnChanges {
...
 
+  private totalCount() : number{
+    let count = 0;
+    this.cartList.forEach((item) => {
+      count += item.quantity 
+    });
+    return count;
+  }
+
+  private totalPrice() : number{
```
Comment (DeimosA) [inline frontend/src/app/shopping-cart-list/shopping-cart-list.component.ts]: Price calculation must account for sale

File: frontend/src/app/shopping-cart/shopping-cart.component.html [hunk 5]
```diff
@@ -13,5 +15,6 @@ <h2>Your shopping cart</h2>
   <div>
     Total price: kr {{ totalPrice() }}
```
Comment (DeimosA) [inline frontend/src/app/shopping-cart/shopping-cart.component.html]: toFixed

**Review by DeimosA — CHANGES_REQUESTED** [review]
see comments

**Comment by arwassa** [comment]
What does toFixed do?

**Comment by DeimosA** [comment]
.toFixed(2) fikserer til 2 decimaler

**Review by DeimosA — APPROVED** [review]

**Pull request merged**